datastore/google/cloud/datastore/client.py

```diff
@@ -511,6 +511,10 @@ def batch(self):
         """Proxy to :class:`google.cloud.datastore.batch.Batch`."""
         return Batch(self)
 
+    def read_only_transaction(self):
```

Comment:
Please remove this factory and add `**kwargs` to the `Client.transaction()` factory. In addition, you might update that docstring to just say the same arguments accepted by `Transaction()` are accepted as keywords.